src/Symfony/Component/Mime/composer.json

```diff
@@ -26,13 +26,14 @@
         "symfony/dependency-injection": "^5.4|^6.0",
         "symfony/property-access": "^5.4|^6.0",
         "symfony/property-info": "^5.4|^6.0",
-        "symfony/serializer": "^5.4|^6.0"
+        "symfony/serializer": "^6.1.6"
```

Comment:
Should we bump dependencies in the `composer.json` or make the changes on 5.4?

Reply:
let's backport on 5.4, that'd prevent anyone from relying on these keys by mistake